src/app/_services/api/import-export.service.ts

```diff
@@ -0,0 +1,115 @@
+import {Injectable} from '@angular/core';
+import {ApiService} from "@app/_services/api.service";
+import {HttpClient, HttpParams} from "@angular/common/http";
+import {Observable} from "rxjs";
+import {Category, Question} from "@app/_models";
+import {catchError} from "rxjs/operators";
+import {ToastrService} from "ngx-toastr";
+import {QuestionService} from "@app/_services/api/question.service";
+import {CategoryService} from "@app/_services/api/category.service";
+
+@Injectable({
+    providedIn: 'root'
+})
+export class ImportExportService {
+    private categories: Category[];
```

Comment:
Don't do this. There should be a way to filter by category id. If there is not add it in the back-end.